NOTICE

```diff
@@ -0,0 +1,7 @@
+                           Apache Comdev
+              Copyright 2020 The Apache Software Foundation
+
+================================================================================
+
```

Comment:
The NOTICE file should not have any extraneous text -- e.g. the === line is not required.
Also the 1st two lines should be left-justified:

http://www.apache.org/legal/src-headers.html#notice

Reply:
Thanks @sebbASF! Just updated it.

Reply:
The page also mentions: http://www.apache.org/legal/src-headers.html#faq-webpages

```
Does this policy also apply to content displayed on our web sites?

No. Our web sites do not have an associated NOTICE file. Instead we 
may soon be making the terms of such content explicit through a "Terms of Use" 
or "Legal Information" link in the footer of web pages. At this point, 
no action is required for Apache web sites.
```

I'll remove the NOTICE file.

Reply:
Not sure that's correct.

There is a difference between the published website, and the source code that is used to create it. The public repo that creates the website may still need the NOTICE file.

Reply:
Hm, should we even have a NOTICE file next to the website sources as normally the website sources are not released => `Every Apache distribution should include a NOTICE file in the top directory, along with the standard LICENSE file`. Website sources are not released as Apache distribution through ASF infrastructure as far as I know.

But of course I can be wrong about this.

If we are indeed planning to release the website sources, then a NOTICE file is definitely needed. 
I can open a LEGAL ticket to discuss whether it is needed for the current form? I was searching through LEGAL tickets but couldn't find anything.

Reply:
AIUI all public ASF code repos should have N&L files - it does not matter if the code is formally released or not. A publicly accessible code repo effectively publishes the code, whether formally released or not.

Reply:
Okay, re-added it. Thanks for explaining it Sebb.